.circleci/config.yml

```diff
@@ -44,9 +44,31 @@ jobs:
               key: figgy-npm-{{ checksum "package-lock.json" }}
               paths:
                 - npm_modules
+          - run: NODE_ENV=test bundle exec rails webpacker:compile
+          - persist_to_workspace:
+              root: /root/figgy
+              paths: '*'
+    test:
+      working_directory: ~/figgy
+      docker:
+        - image: circleci/ruby:2.3.6-node-browsers
+          environment:
+            RAILS_ENV: test
+            FIGGY_DB_HOST: localhost
+            FIGGY_DB_USERNAME: figgy
+            FIGGY_DB_PASSWORD:  ""
+        - image: postgres:9.5
```

Comment:
Do we want to start using postgres 10 for CI?

Reply:
Sure, if I could get CircleCI to run - it stopped running my builds. >.>